WorkcraftCore/src/org/workcraft/util/FileUtils.java

```diff
@@ -70,15 +70,10 @@ public static void copyFileToStream(File in, OutputStream out) throws IOExceptio
         FileInputStream is = new FileInputStream(in);
         FileChannel inChannel = is.getChannel();
         WritableByteChannel outChannel = Channels.newChannel(out);
-        try {
-            inChannel.transferTo(0, inChannel.size(), outChannel);
-        } catch (IOException e) {
-            throw e;
-        } finally {
-            if (is != null) is.close();
-            if (inChannel != null) inChannel.close();
-            if (outChannel != null) outChannel.close();
-        }
+        inChannel.transferTo(0, inChannel.size(), outChannel);
+        if (is != null) is.close();
```

Comment:
I think the original code is correct... The `*.close();` statements will not be executed in the modified code if `inChannel.transferTo` throws an exception. In the original code the channels would be properly closed though.